### PR TITLE
Refactoring CI to use librecores-ci-openrisc docker image

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
This PR resolves the issue of installing the or1k-tests again and again for every Job by using the librecores-ci-openrisc docker image https://github.com/librecores/docker-images/tree/master/librecores-ci-openrisc
CC @stffrdhrn @oleg-nenashev

### CODE REVIEW

File: .travis/test.sh [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 export HOME=/tmp
 
-cd /src
-echo "Installing test scripts"
-./.travis/install-or1k-tests.sh
+# Setup fusesoc and add the cores required by or1k-tests
+fusesoc init -y
+fusesoc library add mor1kx-generic https://github.com/stffrdhrn/mor1kx-generic.git
+fusesoc library add intgen https://github.com/stffrdhrn/intgen.git
+fusesoc library add mor1kx /src
```
Comment (stffrdhrn) [inline .travis/test.sh:10]: Why mor1kx?

File: .travis/test.sh [hunk 2]
```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 
-apt-get update && apt-get install -y curl
-
-export TRAVIS_BUILD_DIR=/src
 export HOME=/tmp
```
Comment (stffrdhrn) [inline .travis/test.sh:3]: Why hardcode home to tmp? If it's needed maybe it should be in the docker image?

**Comment by Nancy-Chauhan** [comment]
@stffrdhrn @oleg-nenashev  I have added documentation !

File: .travis/test.sh (hunk 1 above)
Comment (Nancy-Chauhan) [reply .travis/test.sh:10]: Added : fusesoc library add or1k_marocchino /src

File: .travis/test.sh (hunk 2 above)
Comment (Nancy-Chauhan) [reply .travis/test.sh:3]: @stffrdhrn https://github.com/librecores/docker-images/blob/master/librecores-ci-openrisc/Dockerfile#L6 , I have added it already in dockerfile , but without export HOME=/tmp in test.sh , test seems to fail , as if it doesn't pick that from dockerfile .

**Comment by stffrdhrn** [comment]
It looks like this changed, can you resolve the conflicts? @Nancy-Chauhan

**Comment by Nancy-Chauhan** [comment]
I have resolved the conflicts @stffrdhrn

**Comment by Nancy-Chauhan** [comment]
@stffrdhrn The Build is in the queue for a day. Do we need to retrigger it? I will send PR for yosys in this repo afterwards

**Comment by stffrdhrn** [comment]
I reran. It looks good now

**Pull request merged**